static/lifecycle.svg

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
+ "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<!--
```

Comment:
Nit: To build the SVG in the build script, would you want to use `sed -i` (OS X needs a bit of [finagling](https://unix.stackexchange.com/a/131940) for `sed -i`) to inject the license looking for the string `Generated by graphviz version` for now? Or I could write an XSLT to be a bit more pedantic if you'd like?

Reply:
Ahh, tricky. I couldn't think of a way to do it via my normal text-replacement tools, but your sed trick sounds good to me.

If you have it handy, i'm happy to pull it in, else I'll try to come up with something myself.